src/main/java/com/devonfw/devcon/modules/devon4j/migrate/Migrations.java

```diff
@@ -113,7 +113,25 @@ public static Migrator devon4j(Output output) {
             "implements $1Dao, io.oasp.module.jpa.common.base.LegacyDaoQuerySupport<$1Entity>",
             FileFilterPattern.reject("Application(MasterData)?DaoImpl\\.java")) //
         .and() //
-        .next().build();
+        .next().to(VersionIdentifier.ofDevon4j("3.1.0")) //
+        .pom().replaceProperty("devon4j.version", "3.1.0") //
+        .replaceProperty("spring.boot.version", "2.1.4.RELEASE") //
```

Comment:
I have removed Spring boot version 2.1.4.RELEASE because it is breaking current functionality of devon4j App .need to Upgrade devon4j Spring boot version first.